challenges/objects-arrays.js

```diff
@@ -42,24 +49,43 @@ const graduates = [{"id":1,"first_name":"Cynde","university":"Missouri Southern
 {"id":9,"first_name":"Michail","university":"Universidad Católica de Ávila","email":"mrome8@shinystat.com"},
 {"id":10,"first_name":"Hube","university":"Universitat Rovira I Virgili Tarragona","email":"hlethbrig9@foxnews.com"}]
 
-/* Request 1: Create a new array called universities that contains all the univeristies in the graduates array.  
+/* Request 1: Create a new array called universities that contains all the univeristies in the graduates array.
 
 Once you have the new array created, sort the universities alphabetically and log the result. */
 const universities = [];
+
+for (index = 0; index < graduates.length; index++) {
+    universities.push(graduates[index].university);
```

Comment:
you can just use i to speed up your typing. 👍  But good For loop regardless!

Reply:
thanks, man - typing out 'index' in it's entirety this day was sort of done to explaining to myself what I was doing at the time   haha